husky_components/tabs/main.js

```diff
@@ -15,6 +15,7 @@
  *      - data: if no url is provided
  *      - selected: the item that's selected on initialize
  *      - instanceName - enables custom events (in case of multiple tabs on one page)
+ *      - isTabChangeAllowed: funtion that checks if tab change is allowed
```

Comment:
also why not call this `tabChangeCallback`? 
`isTabChangeAllowed` sounds lika a boolean to me :)